project_2_image_captioning_project/DEVELOPERS.md

```diff
@@ -51,6 +51,8 @@ When done with the Setup and Installation process it time to test our project an
 
 ## Contribution
 If you’re interested in the project, feel free to open an issue, create a PR, or just come say hi and tell us more about yourself.
+
+`Please Read : ` [Code of conduct](code-of-conduct.md)
```

Comment:
Add this in the README.
Don't wrap the Please read in ``

Reply:
done